pkg/postgres/pgadmin/admin_utils.go

```diff
@@ -278,12 +384,27 @@ func GetRemainingCapacity(postgresConfig *pgxpool.Config) (int64, error) {
 
 	row := connectPool.QueryRow(ctx, totalSizeStmt)
 	var sizeUsed int64
-	if err := row.Scan(&sizeUsed); err != nil {
+	err := row.Scan(&sizeUsed)
+	if err != nil {
 		return 0, err
 	}
 
-	log.Infof("size used = %d.  size left = %d", sizeUsed, pgconfig.GetPostgresCapacity()-sizeUsed)
-	return pgconfig.GetPostgresCapacity() - sizeUsed, nil
+	var capacity int64
+	if env.ManagedCentral.BooleanSetting() {
+		// Cannot get managed services capacity via Postgres.  Assume size for now.
+		capacity = pgconfig.GetPostgresCapacity() - sizeUsed
+	} else {
+		capacity, err = getAvailablePostgresCapacity(postgresConfig)
+		if err != nil {
+			log.Error(err)
+			// If we cannot calculate the capacity, assume it based on the recommended starting
+			// capacity
+			return pgconfig.GetPostgresCapacity() - sizeUsed, err
+		}
+	}
+
+	log.Infof("remaining capacity = %d", capacity)
```

Comment:
It is better to differentiate the results for managed central in log?

Reply:
I'm not sure what you are meaning exactly.  But I wouldn't think that would be necessary in the log unless we want to put another caveat saying it is based off a value for now.  We will know that a managed central is using a managed Postgres at least initially.

Reply:
I mean to put another (different) log within
`if env.ManagedCentral.BooleanSetting() {
}
`
so that we know if the capacity here is an estimate or not.